components/ILIAS/Chatroom/resources/js/src/Type.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+const HEARTBEAT_TIMEOUT = 5000;
+
+export class TypeSelf {
```

Comment:
Can we add some more context to the class-names and the filename?

Reply:
I added doc blocks. Please have a look if this is what you had in mind.